cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java

```diff
@@ -1335,12 +1332,6 @@ private boolean tryPullTimeSeriesSchemas(Node node, PullSchemaRequest request) {
    * @return all paths after removing wildcards in the path
    */
   public List<PartialPath> getMatchedPaths(PartialPath originPath) throws MetadataException {
-    // make sure this node knows all storage groups
-    try {
-      metaGroupMember.syncLeaderWithConsistencyCheck(false);
-    } catch (CheckConsistencyException e) {
-      throw new MetadataException(e);
-    }
```

Comment:
Hi, @LebronAl I see there are so many places use this method, how to make sure that this modification has no effect on the other logic that calls this function?

The same as other methods.

Reply:
Manually review their call stacks one by one to ensure that a query request only performs synchronization once.

Maybe we need to analyse the current synchronization of CMManager and fix it further.

Reply:
Ok